OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -202,6 +325,7 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
         String VIDEO_DOWNLOADED = "edx.bi.video.downloaded";
         String USERLOGOUT = "edx.bi.app.user.logout";
         String USERLOGIN = "edx.bi.app.user.login";
+        String USER_REGISTER = "edx.bi.app.user.register";
```

Comment:
Is this possibly a duplicate of `String CREATE_ACCOUNT_CLICK = "edx.bi.app.user.register.clicked";
`